include/macros.h

```diff
@@ -64,4 +64,6 @@
 // aligns an address to the next 16 bytes
 #define ALIGN16(val) (((val) + 0xF) & ~0xF)
 
+#define SQUARE(x) ((x) * (x))
```

Comment:
I'm not sure if this is the right header for this.

How do you feel about putting it in mk64.h instead?

Reply:
Ok